workflows/src/main/kotlin/com/template/flows/ApproveAppointmentRequest.kt

```diff
@@ -62,28 +72,20 @@ class ApproveAppointmentRequest(private val alice: Party,
 
         // Step 6. Collect the other party's signature using the SignTransactionFlow.
```

Comment:
This comment is outdated, you are not collecting signatures here. You can delete all those comments from the template or keep them up to date.